docs: Add YugabyteDB smart driver features to Sequelize documentation (#29496)
yugabyte/yugabyte-db: docs/content/stable/develop/drivers-orms/nodejs/sequelize.md

- The actual table is created by calling the `Employee.sync()` API in the `createTableAndInsert()` function. This also inserts the data for three employees into the table using the `Employee.create()` API.
- Finally, you can retrieve the information of all employees using `Employee.findAll()`.

> #### Using the YugabyteDB Smart Driver

Reviewer: ```suggestion
#### Using the YugabyteDB node-postgres smart driver
```

You can also use the YugabyteDB smart driver (`@yugabytedb/pg`) which provides connection load balancing feature that spreads connections uniformly across the nodes in the entire cluster or in specific placements (zones or regions).

> Read more on the smart driver features [here](../../smart-drivers/#using-yugabytedb-smart-drivers).

Reviewer: ```suggestion
For more information on the smart driver features, refer to [Using YugabyteDB smart drivers](../../smart-drivers/#using-yugabytedb-smart-drivers).
```

> To use the YugabyteDB smart driver instead of the vanilla PG driver, add the following to your application's `package.json` dependencies:

Reviewer: ```suggestion
To use the YugabyteDB node-postgres smart driver instead of the vanilla pg driver, add the following dependencies to your application's `package.json` file, to override the default PostgreSQL driver with the YugabyteDB smart driver:
```

]
```

#### Specify load balance properties

You can specify load balance properties using environment variables or connection parameters.

For information on all load balance modes, see [Node type-aware load balancing](../../smart-drivers/#node-type-aware-load-balancing).

##### Environment variables

> The following table summarizes the environment variables available for configuring the YugabyteDB node-postgres smart driver:

Reviewer 2: @ashetkar These Env variables are common to the driver not just sequelize? Why is this table not here https://docs.yugabyte.com/stable/develop/drivers-orms/nodejs/yugabyte-node-driver/#step-2-set-up-the-database-connection

]
```

#### Specifying load balance properties

You can enable these features by specifying the properties either as the environment variables or via connection parameters.

##### Environment variables

```js
const { Sequelize, DataTypes } = require('sequelize-yugabytedb')

// Enable load balancing across nodes in the cluster
process.env.PGLOADBALANCE = 'any';  // Valid values: any, prefer-primary, prefer-rr, only-primary, only-rr
// Specify the region(s)/zone(s) to target nodes from (Optional)
process.env.PGTOPOLOGYKEYS = 'aws.us-east-2.us-east-2a';
// Set the minimum time interval for refreshing the cluster topology information (Optional)
process.env.PGYBSERVERSREFRESHINTERVAL = '5';

const sequelize = new Sequelize('yugabyte', 'yugabyte', 'yugabyte', {
   host: 'localhost',
   port: '5433',
   dialect: 'postgres'
})
```

> 

Reviewer: ```suggestion
For information on all load balance modes, see [Node type-aware load balancing](../../smart-drivers/#node-type-aware-load-balancing).
```
